### PR TITLE
Tox21 multi task

### PR DESCRIPTION
**Pull request by nd-02110114**
This PR is mainly fixing tutorials.

What I did
- remove the clipped_sigmoid function 
- rename `nn` directory to `readout`
- create loss directory and put on `binary_cross_entropy_with_logits` function
- change single task classification to the multi task classification about the Tox21 tutorial
- change file name about Tox21 tutorial (scripts and notebook)

### CODE REVIEW

**Comment by nd-02110114** [comment]
@rbharath  
Thanks!
